doc/TYPES2021/Demos/HSP.tex

```diff
@@ -11352,6 +11347,14 @@ \subsection{Formal proof}
 
 This completes the formal proof of Birkhoff's variety theorem.
 
+%% -----------------------------------------------------------------------------
+\section{Discussion}\label{sec:discuss}
+How do we differ from the classical, set-theoretic approach? Most noticeable is
+our avoidance of all \emph{size} issues. By using universe levels and level
+polymorphism, we always make sure we are in a \emph{large enough} universe.
```

Comment:
Nice!  (although, you might want to drop "always" ...it doesn't quite fit our terse style) :smile: